lms/djangoapps/branding_stanford/models.py

```diff
@@ -4,10 +4,13 @@
 This was implemented initially so that the list of course tiles could be
 stored in this model.
 """
+from django.conf import settings
 from django.db import models
 
 from config_models.models import ConfigurationModel
 from opaque_keys.edx.django.models import CourseKeyField
+from opaque_keys.edx.keys import CourseKey
```

Comment:
With these imports side-by-side, it's more obvious this should have been changed when `CourseKeyField` was too.